test/test.js

```diff
@@ -405,10 +405,10 @@ test('runner have test event', function (t) {
 		a.end();
 	});
 
-	runner.on('test', function (err, title, duration) {
+	runner.on('test', function (err, props) {
 		t.error(err);
-		t.equal(title, 'foo');
-		t.ok(duration !== undefined);
+		t.equal(props.title, 'foo');
```

Comment:
Just fyi, tape supports `t.is` as an alias, so we could use that. Probably better as a separate PR though.